dot/utils_test.go

```diff
@@ -43,3 +46,43 @@ func TestNewConfigAndFile(t *testing.T) {
 	require.NotNil(t, testCfg)
 	require.NotNil(t, testCfgFile)
 }
+
+// TestInitNode
+func TestNewTestGenesis(t *testing.T) {
+	cfg := NewTestConfig(t)
+	require.NotNil(t, cfg)
+
+	genFile := NewTestGenesisFile(t, cfg)
+	require.NotNil(t, genFile)
+
+	defer utils.RemoveTestDir(t)
+
+	cfg.Init.Genesis = genFile.Name()
+	fmt.Printf("FileName %v\n", genFile.Name())
+}
+
+func TestNewTestGenesisHRFile(t *testing.T) {
+	cfg := NewTestConfig(t)
+	require.NotNil(t, cfg)
+
+	genHRFile := NewTestGenesisHRFile(t, cfg)
+	require.NotNil(t, genHRFile)
+	defer os.Remove(genHRFile.Name())
+
+	genRawFile := NewTestGenesisFile(t, cfg)
+	require.NotNil(t, genRawFile)
+	defer os.Remove(genRawFile.Name())
+
+	genHR, err := genesis.NewGenesisFromJSONHR(genHRFile.Name())
+	require.NoError(t, err)
+	genRaw, err := genesis.NewGenesisFromJSON(genRawFile.Name())
+	require.NoError(t, err)
+
+	// values from raw genesis file should equal values generated from human readable genesis file
+	//  Note, we don't just compare Genesis.Raw because there are entries (balances) in raw genesis that are not yet handled by
```

Comment:
feel free to remove the balances from `genesis.json` since the keys have changed since the runtime was updated